crates/ruff_python_ast/Cargo.toml

```diff
@@ -24,4 +24,5 @@ regex = { workspace = true }
 rustc-hash = { workspace = true }
 rustpython-common = { workspace = true }
 rustpython-parser = { workspace = true }
+serde = { workspace = true }
```

Comment:
Why did we have to add the `serde` dependency?

Reply:
Sorry, I should've documented this. I think it's an issue with how I added the `serde` feature in RustPython. It looks like it doesn't have the `derive` feature enabled. So we need to use "our" `serde`, which _does_ have `derive`, in order for it to compile. (This is based on cursory guess, I didn't look deeply, but I'll create an issue and document.)